providers/azure/resources/cloud_defender.go

```diff
@@ -454,19 +454,21 @@ func (a *mqlAzureSubscriptionCloudDefenderService) securityContacts() ([]interfa
 	if err != nil {
 		return nil, err
 	}
+	res := []interface{}{}
 	list, err := getSecurityContacts(ctx, armConn)
 	if err != nil {
-		return nil, err
+		// https: //github.com/mondoohq/cnquery/issues/4997
+		log.Warn().Err(err).Msg("fail gracefully")
+		return res, nil
 	}
-	res := []interface{}{}
 	for _, contact := range list {
 		alertNotifications, err := convert.JsonToDict(contact.Properties.AlertNotifications)
 		if err != nil {
-			return nil, err
+			log.Debug().Err(err).Msg("unable to convert armsecurity.Contact.Properties.AlertNotifications to dict")
 		}
 		notificationsByRole, err := convert.JsonToDict(contact.Properties.NotificationsByRole)
```

Comment:
do we need to check if contact or contact.Properties are set?

Reply:
yeah, we probably should have done that, I'll add  the check.